adhocracy4/polls/api.py

```diff
@@ -178,6 +179,8 @@ def vote(self, request, pk):
             self.save_vote(question, vote_data, creator, content_id)
 
         poll = self.get_object()
+        print("##########################################")
+        poll_voted.send(sender=None, poll=poll, creator=creator, content_id=content_id)
```

Comment:
sender is the poll view, I have added it as the `self.__class__` in my PR for mB  kiezradar.

Reply:
and the print statement can go I believe :)

Reply:
thanks, added